### PR TITLE
update layout to be more flexy

### PR DESCRIPTION
**Pull request by spasovski**
- makes things work better on mobile
- removes `position: absolute` from filters
- creates sign-in wrapper for the upcoming auth component

### CODE REVIEW

File: distributionviewer/core/static/css/distributionviewer.styl [hunk 1]
```diff
@@ -9,62 +9,37 @@ body {
...
 }
 
 a {
   color: $mild-blue;
 }
 
 .global-wrapper {
+  display: flex;
+  flex-wrap: wrap;
   margin: 0 auto;
```
Comment (openjck) [inline distributionviewer/core/static/css/distributionviewer.styl]: We won't need this with a `100%` width.

File: distributionviewer/core/static/css/distributionviewer.styl [hunk 2]
```diff
@@ -9,62 +9,37 @@ body {
...
 .filters {
+  display: none;
   font-weight: 100;
   height: 100%;
-  left: 0;
-  padding: $header-height 20px 20px 0;
-  position: absolute;
-  top: 0;
+  padding: 40px 20px 20px 0;
+  margin-left: -10px;
```
Comment (openjck) [inline distributionviewer/core/static/css/distributionviewer.styl]: What's the negative margin for?

**Comment by openjck** [comment]
Added a couple of line comments. There are also at least two variables in _lib.styl_ that we can remove now. Otherwise this looks great!

File: distributionviewer/core/static/css/distributionviewer.styl (hunk 2 above)
Comment (spasovski) [inline distributionviewer/core/static/css/distributionviewer.styl]: To ignore the `.global-wrapper` padding (hover effect should touch the left edge). I should leave a comment though.

File: distributionviewer/core/static/css/distributionviewer.styl (hunk 1 above)
Comment (spasovski) [inline distributionviewer/core/static/css/distributionviewer.styl]: Good catch!

File: distributionviewer/core/static/css/distributionviewer.styl (hunk 2 above)
Comment (openjck) [inline distributionviewer/core/static/css/distributionviewer.styl]: Gotcha. We could use a variable if we know they should always be in sync.

File: distributionviewer/core/static/css/distributionviewer.styl (hunk 1 above)
Comment (openjck) [inline distributionviewer/core/static/css/distributionviewer.styl]: Looks like this is still here. :smile:

r+wc so feel free to self-merge

Comment (spasovski) [inline distributionviewer/core/static/css/distributionviewer.styl]: Hah damnit yea my mind if fried today too.

**Pull request merged**